src/layout/footers/footer.js

```diff
@@ -29,7 +29,7 @@ const Footer = () => {
                 </p>
                 <ul className="list-wrap footer__social">
                   <li>
-                    <a href="https://www.linkedin.com/">
+                    <a href="https://www.linkedin.com/company/greenbyte-io/" target="_blank">

```

Comment:
Please add this rel="noopener noreferrer"

Reply:
done